app/Http/Controllers/Frontend/AuthController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace HotelBooking\Http\Controllers\Frontend;
+
+use Request;
+use HotelBooking\Http\Controllers\Controller;
+use Auth;
+use HotelBooking\Http\Requests\Frontend\LoginRequest;
+use Session;
+use Route;
+use Response;
+
+/**
+ * Auth Controller for frontend users
+ */
+class AuthController extends FrontendBaseController
+{
+    /**
+     * Authenticate object.
+     */
+    protected $auth;
+
+    /**
+     * Controller constructor.
+     */
+    public function __construct()
+    {
+        $this->auth = Auth::user();
+        $this->middleware('user.guest', ['except' => 'getLogout']);
+    }
+
+    /**
+     * Check log in data.
+     */
+    public function postLogin(LoginRequest $request)
+    {
+        $data = $request->only('username', 'password');
+        $login = $this->auth->attempt($data);
+        if ($login) {
+            $redirectUrl = $request->input('pathname');
```

Comment:
redirectUrl shoud be the previous url https://laracasts.com/discuss/channels/general-discussion/laravel-5-referrer-url.
if there is no previous url then redirect to home page